### PR TITLE
Revise carousel animation

### PR DESCRIPTION
**Pull request by khyechun**
Card: [Carbon Website - Revise Carousel animation](https://www.notion.so/switcheo/Carbon-Website-Revise-Carousel-animation-46fe2fcb27164fc8b2d14f3ee0c6bd0d)

- Added opacity transition for background image
- Changed background image position
- Updated size transition animation

https://github.com/Switcheo/carbon-website/assets/47170358/749a2efb-178f-42ff-8c2f-686911ffd1d6

### CODE REVIEW

File: src/@carbon-info/views/Ecosystem/components/AppsCarousel.tsx [hunk 1]
```diff
@@ -35,32 +35,26 @@ const AppsCarousel: React.FC<Props> = (props: Props) => {
...
+        return <Grow in timeout={(index + 1) * 200 > 1000 ? 1000 : (index + 1) * 200} key={`${name}-featured-dApps`} >
+          <Box id={`list-item-${index}`} className={clsx(classes.cardContainer, index === view && "expandCard", { open: inView }, { lastCard: index === (view + preview) % totalCards })}>
+            <Typography variant="body1" color="textPrimary" className={classes.tag}>{tag}</Typography>
+            <img src={icon} className={index === view ? classes.dAppLogo : classes.logo} />
+            <Typography variant="h3" color="textPrimary" className={classes.nameLabel}>{name}</Typography>
+            {index === view && <Typography variant="body1" style={{ color: theme.palette.text.button, marginTop: "1rem" }}>{description}</Typography>}
+            <Button className={classes.ctaButton} href={ctaLink} target="_blank">{buttonText}</Button>
+            <Box className={clsx(classes.backgroundImage, view !== index && 'inactive')}  style={{ backgroundImage: `url("${item?.backgroundImage}")` }} />
+          </Box>
+        </Grow>
```
Comment (sarah-thong) [inline src/@carbon-info/views/Ecosystem/components/AppsCarousel.tsx:57]: Do add a bracket around rendered elements.
i.e.
```typescript
return (
  <Grow in timeout={(index + 1) * 200 > 1000 ? 1000 : (index + 1) * 200} key={`${name}-featured-dApps`} >
    {...}  
  </Grow>
)
```

**Comment by sarah-thong** [comment]
### UI Comments
The movement when clicking the left button looks great and smooth, but the movement when clicking the right button looks quite awkward (i.e. main card enlarges while pushing the other elements back).

Atm, I can think of 2 solutions:
1. Make the side padding enlargement instant (i.e. 5rem right and left padding instantly), then then slowly make enlarge the top + bottom padding (i.e. 5rem top and bottom over 0.7s).
2. Reduce the pushing back effect (e.g. on expandCard, make side padding 4rem, then enlarge it to 5rem over 0.7s)

**Review by sharmainec — COMMENTED** [review]
Have this weird issue with the second slide border on safari (border is too tall for the slide) and third slide (bottom and top border not visible)
- I don't have this issue on chrome

image

File: src/@carbon-info/views/Ecosystem/components/AppsCarousel.tsx [hunk 2]
```diff
@@ -222,6 +212,29 @@ const useStyles = makeStyles((theme: Theme) => ({
     fontWeight: 700,
     marginTop: "1rem",
   },
+  itemContainer: {
+    height: '100vh',
```
Comment (sharmainec) [inline src/@carbon-info/views/Ecosystem/components/AppsCarousel.tsx:216]: might need to reduce the height as the current gap between the header and the carousel is a bit too much

image

File: src/@carbon-info/views/Ecosystem/components/AppsCarousel.tsx [hunk 3]
```diff
@@ -222,6 +212,29 @@ const useStyles = makeStyles((theme: Theme) => ({
...
+    display: 'flex',
+    alignItems: 'center',
+    [theme.breakpoints.down('sm')]: {
+      height: '55vh',
```
Comment (sharmainec) [inline src/@carbon-info/views/Ecosystem/components/AppsCarousel.tsx:220]: also need to reduce the height for mobile to reduce the gap

image

File: src/@carbon-info/views/Ecosystem/components/AppsCarousel.tsx (hunk 2 above)
Comment (khyechun) [reply src/@carbon-info/views/Ecosystem/components/AppsCarousel.tsx]: Replaced the use of VH values, so this should be the same on all screens now.

**Pull request merged**